### DISCUSSION TITLE
Use-Case Discussion

### DISCUSSION THREAD

**Discussion by egecans** [bounswe/bounswe2023group2#102] [Ideas]
I open this for discussing and asking things that make you confused about the use-case diagram. If someone works parallel with you on lucid, the first thing that you should do is follow his/her cursor and ask your question to him/her. If s/he is also confused about the situation or if you are working alone at that time, you can ask your question here to discuss it.

**Comment by egecans** [comment]
I painted extends lines purple which are concentric includes lines. I did that for separating them easily, but I don't know whether it's correct or not.

**Comment by egecans** [comment]
Can you look profile? I separate view other user's profiles and own profile because of their scopes. But I keep the older version in diagrams. You can pick which of them is suitable for our case.

**Reply by bgmrsln** [reply]
I guess it looks more clear this way. But I guess you did not add others into use case diagram, right?

**Comment by mehmetkuzu** [comment]
Profile is ok as you did. 
But two details:
1. I think edit mandatory info should not be an extend to edit user. It is an include, even edit info itsel. (Others are extends as they are optional) - I will make the change to show the idea. We can change back if necessary.
2. The containers. All of the containers in the diagram should have the same label: Disaster response and recovery system.

**Reply by mehmetkuzu** [reply]
-- When making the change, I decided to show the mandatory fields as direct connection to edit info (not includes) as they actually are the input fields in the edit info

**Reply by egecans** [reply]
- Yes, a user doesn't have to edit mandatory fields whenever s/he enters the edit profile part, therefore it shouldn't be included.
- I think containers are suitable to indicate scopes. Maybe they can be included in the Disaster response and recovery system at the end. As like activity and need containers which I made now.

**Comment by egecans** [comment]
I changed the filter&sort, the old version is still available on the diagram, you can use which one is suitable.

**Comment by egecans** [comment]
I think I finished the use-case diagram excepting arrows from users. Maybe the View main page could be appended to it, but I think it's system requirements. Users can see the main page after they are logged in. But they can return back to the main page maybe from their profiles or maps. It's your choice, but don't forget that guests, registered users and the admin's main page are different from each other. If you think it's redundant don't need waste time on that, while we haven't finished class and sequence diagrams.

**Comment by egecans** [comment]
Since milestone 1 was over I close this discussion.
